### ISSUE TITLE
selector-combinator-space-before conflict with Prettier

### ISSUE THREAD

**Issue by henrikhermansen** [shannonmoeller/stylelint-config-prettier#5]
**Dependencies:**
"prettier": "1.9.2",
"stylelint-config-standard": "^18.0.0",
"stylelint-config-prettier": "^2.0.0",

**Formatting produced by Prettier:**
```
.someClass {
    &
        + .someClass__someModifier
        > .someClass__anotherModifier {
        color: red;
    }
}
```

**Stylelint errors:**
Expected single space before "+"  selector-combinator-space-before
Expected single space before ">"  selector-combinator-space-before

**Comment by shannonmoeller** [comment]
Will disable. Thanks!

**Comment by shannonmoeller** [comment]
Fixed by 418af4e.
